core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3245,12 +3245,15 @@ private void setAclSingleInode(RpcContext rpcContext, SetAclAction action,
       case REPLACE:
         // fully replace the acl for the path
         targetInode.replaceAcl(entries);
+        targetInode.updateMask(entries);
```

Comment:
why can't updateMask() functionality just be a part of `replaceAcl` or `setAcl`?

Reply:
made it part of those functions.